config/training/default_training.yaml

```diff
@@ -8,7 +8,6 @@ training:
   max_epochs: ${general.max_epochs}
   min_epochs: ${general.min_epochs}
   num_workers:
-  loss_fn: CrossEntropy  # name of the metric you want to manage the training
```

Comment:
Ideally, the loss parameter should stay in the training config, but I couldn't point this parameter to the one in config/loss/multiclass use hydra's ${...} syntax. I did try quite a bit. I'd be moving the loss parameter out of training if we can't find a way to make this work.